.circleci/config.yml

```
@@ -44,6 +44,15 @@ jobs:
       - store_test_results:
           path: test_results
 
+  chromatic: 
```

Let's name this step more generic in case we change tooling in the future
```suggestion
  storybook_preview: 
```